### ISSUE TITLE
Large overlap of "AP" and "CHEST (PA and LAT)" images in MIMIC-CXR

### ISSUE THREAD

**Issue by maximek3** [MIT-LCP/mimic-code#1513]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

Hi, I have found that there is an overlap of 22,043 chest X-rays in MIMIC-CXR that have as `ViewPosition` `AP` but their `PerformedProcedureStepDescription` is given as `CHEST (PA and LAT)`. Are those X-rays all the ones that are `AP`, but not taken with a portable scanner? 

Thanks for your help!

**Comment by maximek3** [comment]
Thanks for getting back on this.

In `ProcedureCodeSequence_CodeMeaning` it seems that `CHEST (PA and LAT)` and `CHEST (PORTABLE AP)` make up >99% of the possible values. 

Would it be fair to assume that whenever a scan is `AP` and `CHEST (PA and LAT)`, it's an AP scan taken with a fixed scanner? And when it's `AP` and `CHEST (PORTABLE AP)`, it's taken with a portable scanner?

**Comment by alistairewj** [comment]
That's what I would do. And then I'd check ~10-20 records to make sure it's consistent with what I see. For a while we had a model which classified images as PA or AP, since it's a very easy classification problem, so that's an alternative option.

**Comment by maximek3** [comment]
Thanks for the pointers, I did this, and here is what I found:

For 20 studies that have both `AP` and `CHEST (PA and LAT)`:

- 3 mention in their report they are PA scans. But their `viewpoint` is given as `AP`. The study ids are 55653823, 58957089, and 59686004. However, when looking at the images, at least for 55653823 and 59686004, it becomes clear that it is actually an AP image (it's marked on the scan), so it seems the comment in the report is wrong, and the `viewpoint` from the meta file is correct.
- 6 are described as "frontal" in the report.
- The remaining 11 are described as "AP" in the report.

For 10 studies that have `AP` and `CHEST (PORTABLE AP)`, 8 clearly indicate in the report that the scans were taken on a portable scanner. For the remaining 2 there is no information excluding this.

All in all, it does seem like the assumptions below is correct :) 

> Would it be fair to assume that whenever an image is both AP and CHEST (PA and LAT), it's an AP scan taken with a fixed scanner? And when it's AP and CHEST (PORTABLE AP), it's an AP scan taken with a portable scanner?
